netlogo-headless/src/main/prim/etc/_shuffle.scala

```diff
@@ -4,20 +4,20 @@ package org.nlogo.prim.etc
 
 import java.util.{ Collections, List => JList }
 import scala.collection.mutable.MutableList
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
 
 import org.nlogo.core.LogoList
 import org.nlogo.nvm.{Reporter, Context}
 
 class _shuffle extends Reporter {
   def report(context: Context): AnyRef = {
-    val result: JList[AnyRef] = scala.collection.mutable.MutableList(argEvalList(context, 0): _*)
+    val result: JList[AnyRef] = scala.collection.mutable.MutableList(argEvalList(context, 0).scalaIterator.toSeq: _*).asJava
```

Comment:
I think the entire sequence gets copied twice here: once from `LogoList` to `Stream`, then again from `Stream` to `MutableList`. Surely it's possible to get away with only one copy?